src/js/actions/confirm-email.js

```diff
@@ -0,0 +1,43 @@
+export const SEND_EMAIL_REQUEST = 'SEND_EMAIL_REQUEST';
```

Comment:
I will change the name of actions to CONFIRM_EMAIL_REQUEST , as this makes more sense